src/components/Sponsors/Sponsors.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styles from '../Sponsors/Sponsors.module.css';
+
+import StarBg from "../../assets/Images/bg_landing_stars.svg";
+import Sponsor from '../../assets/Images/sponsor_logo.png';
+import frame from '../../assets/Images/frame.png';
```

Comment:
After moving the images to public directory, you won't need these imports

Reply:
Done